db/revision_cache_interface.go

```diff
@@ -47,10 +47,10 @@ type RevisionCache interface {
 	Upsert(ctx context.Context, docRev DocumentRevision)
 
 	// Remove eliminates a revision in the cache.
-	Remove(docID, revID string)
+	Remove(docID, revID string, cv *CurrentVersionVector)
 
 	// UpdateDelta stores the given toDelta value in the given rev if cached
-	UpdateDelta(ctx context.Context, docID, revID string, toDelta RevisionDelta)
+	UpdateDelta(ctx context.Context, docID, revID string, cv *CurrentVersionVector, toDelta RevisionDelta)
```

Comment:
I didn't think we should be modifying this API until we support version-based Delta (which would include the required changes to RevisionDelta in this file).